ci/lint-docs.sh

```diff
@@ -3,6 +3,6 @@
 
 set -eux
 
-git ls-files | xargs codespell
+git ls-files | xargs codespell -L keypair,iam,hda
 sphinx-build -M clean doc/ doc/_build
-sphinx-build -nW doc/ doc/_build
+sphinx-build -n doc/ doc/_build
```

Comment:
Do we really need to remove `W`?

Reply:
Yes :/
We have circular imports guarded by `typing.TYPE_CHECKING`. That's not a problem for Mypy but it is a problem for Sphinx.

Reply:
Ok :(